src/freon.c

```diff
@@ -29,6 +29,10 @@
     system("host-dbus dbus-send --system --dest=org.chromium.LibCrosService " \
            "--type=method_call --print-reply /org/chromium/LibCrosService " \
            "org.chromium.LibCrosServiceInterface." #function)
+#define FREON_DBUS_METHOD_CALL_NEW(function) \
```

Comment:
this one can be DISPLAYSERVICE_METHOD_CALL

Reply:
Done.